pkg/env/env.go

```diff
@@ -80,47 +96,96 @@ func ParseFile(path string) (_ map[string]string, err error) {
 	}
 	defer fh.Close()
 
-	scanner := bufio.NewScanner(fh)
-	for scanner.Scan() {
-		// trim the line from all leading whitespace first
-		line := strings.TrimLeft(scanner.Text(), whiteSpaces)
-		// line is not empty, and not starting with '#'
-		if len(line) > 0 && !strings.HasPrefix(line, "#") {
-			if err := parseEnv(env, line); err != nil {
-				return nil, err
-			}
-		}
+	content, err := io.ReadAll(fh)
+	if err != nil {
+		return nil, err
 	}
-	return env, scanner.Err()
+
+	// replace all \r\n and \r with \n
+	text := strings.NewReplacer("\r\n", "\n", "\r", "\n").Replace(string(content))
+	if err := parseEnv(env, text, false); err != nil {
+		return nil, err
+	}
+
+	return env, nil
 }
 
-func parseEnv(env map[string]string, line string) error {
-	data := strings.SplitN(line, "=", 2)
+// parseEnv parse the given content into env format
+func parseEnv(env map[string]string, content string, enforceMatch bool) error {
```

Comment:
Please document what `enforceMatch` does.

Reply:
This variable may not require additional comments to explain its purpose, as it can be inferred from its name: it throws an error if there is no match. This variable is similar to `regexp.MustCompile`.

Reply:
I wouldn't ask if it was _that_ obvious.  To understand the meaning, I had read parts of the code and look at the various callers.  It's not immediately clear.

Reply:
Done 😃

Reply:
Thanks!